core/src/main/java/it/pagopa/selfcare/mscore/core/strategy/CreateInstitutionStrategyAnac.java

```diff
@@ -0,0 +1,74 @@
+package it.pagopa.selfcare.mscore.core.strategy;
+
+import it.pagopa.selfcare.mscore.api.InstitutionConnector;
+import it.pagopa.selfcare.mscore.api.PartyRegistryProxyConnector;
+import it.pagopa.selfcare.mscore.constant.CustomError;
+import it.pagopa.selfcare.mscore.constant.Origin;
+import it.pagopa.selfcare.mscore.core.strategy.input.CreateInstitutionStrategyInput;
+import it.pagopa.selfcare.mscore.exception.MsCoreException;
+import it.pagopa.selfcare.mscore.exception.ResourceConflictException;
+import it.pagopa.selfcare.mscore.model.institution.Institution;
+import it.pagopa.selfcare.mscore.model.institution.SaResource;
+import lombok.extern.slf4j.Slf4j;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+import static it.pagopa.selfcare.mscore.constant.GenericError.CREATE_INSTITUTION_ERROR;
+
+@Slf4j
+public class CreateInstitutionStrategyAnac implements CreateInstitutionStrategy {
+    private final PartyRegistryProxyConnector partyRegistryProxyConnector;
+
+    private final InstitutionConnector institutionConnector;
+
+    private Institution institution;
```

Comment:
you might remove institution object if it is not used

Reply:
Why isn't it used?

Reply:
at row 44 you set institution as a new object returned by addFieldsToInstitution.....you don't use the object passed in constructor